include/slang/syntax/SyntaxNode.h

```diff
@@ -137,6 +138,23 @@ class SLANG_EXPORT SyntaxNode {
     TokenOrSyntax getChild(size_t index);
 };
 
+SyntaxNode* clone(const SyntaxNode&, BumpAllocator&);
```

Comment:
These functions need to be marked SLANG_EXPORT to work with shared library builds.